CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Stache 1.1.3
+
+* Added `compileMustacheDirCustom` that allows custom file extensions such as `.mush`
+* Updated to stackage 9.5, and Megaparsec 6.1.1
```

Comment:
Please drop "Updated to stackage 9.5, and Megaparsec 6.1.1". `stack.yaml` is not a part of the actual package in any way, it's just a file in the repo (for convenience).

----

Also please end the previous point with a full stop (sorry for nitpicking).